databroker/src/broker.rs

```diff
@@ -898,8 +899,8 @@ impl ChangeSubscription {
                     if notifications.updates.is_empty() {
                         Ok(())
                     } else {
-                        match self.sender.send(notifications).await {
-                            Ok(()) => Ok(()),
+                        match self.sender.send(notifications) {
+                            Ok(_) => Ok(()),
```

Comment:
Unless there's a good reason, it's better to match explicitly (so a potential change in the return values of `send`, however unlikely, would be caught by the compiler).

Reply:
The size value returned by `send` method represents the number of currently subscribed Receiver handles at the time of the send, my question is, why is that relevant? how should we handle? just output in `debug`?

Reply:
What is the value range we expect. Like is 0 an expected response, or should it always be 1 or more?

Reply:
After doing a test it seems to be always 1, since each new subscription creates its own broadcast::channel and senders are independent from each other.

Reply:
I guess the return value actually doesn't matter in this case. But when putting it behind a `_` it's not evident what is returned.

> Unless there's a good reason, it's better to match explicitly [...]

I'm ok with ignoring it.